packages/gatsby-codemods/src/transforms/__testfixtures__/gatsby-plugin-image/compat.output.js

```diff
@@ -0,0 +1,6 @@
+import { GatsbyImage } from "gatsby-plugin-image";
+// how do comments work?
+
+<GatsbyImage image={data.file.childImageSharp.gatsbyImageData} alt="headshot" />
+
+//still doing ok?
```

Comment:
I don't know, it's 2020 -- we're just all doing what we can 🤷

Reply:
Lol, this was my "does it handle comments" test